ast35/Python/ast.c

```diff
@@ -1304,10 +1320,16 @@ handle_keywordonly_args(struct compiling *c, const node *n, int start,
                 if (!arg)
                     goto error;
                 asdl_seq_SET(kwonlyargs, j++, arg);
-                i += 2; /* the name and the comma */
+                i += 1 + (TYPE(CHILD(n, i + 1)) == COMMA); /* the name and the comma, if present */
```

Comment:
I'd be inclined to expand all of these to something like
```
i += 1; /* the name */
if (TYPE(CHILD(n, i)) == COMMA)
    i += 1;
```

Couple more lines, but not many more tokens, and I think easier to read and understand to the point of being confident of.